docs/get_started/product_overview/content_management.md

```diff
@@ -5,33 +5,33 @@ tags:
 
-Here is an example of a content template where you can drag & drop and configure any element as part of content.
+The following example of a content template shows how you can drag and configure all elements as part of content.
 ![Overview - CMS](assets/overview-cms.png)
 
-## Headless Content Access
+## Headless content access
 
-Content can be accessed headlessly, using the [WCM REST APIs](api_access.md). This enables content stored in HCL DX to be integrated to applications created outside the platform, for example a mobile application built using HCL MX, a commerce site built using HCL Commerce or a standalone React application.
+The team can access content headlessly by using the [WCM REST APIs](api_access.md). With this API, users can integreate content stored in HCL DX with applications created outside the platform, for example, a mobile application built with HCL MX, a commerce site built with HCL Commerce, or a standalone React application.
 
-## Standalone and Inline Content Editors
+## Standalone and inline content editors
 
-Content can be edited either standalone or inline with a website hosted by HCL DX.
+Users can edit content in a standalone application or inline with a website hosted by HCL DX.
 
-If you are primarily thinking about your content outside of a specific presentation, then you can use our content authoring tools to create and manage individual content items. That content editor is shown below:
+If you are primarily thinking about your content outside of a specific presentation, then you can use the DX content authoring tools to create and manage individual content items. The following figure shows the content editor:
 
 ![Overview - Content Editor](assets/overview-content-editor.png)
 
-On the other hand, many people create content in the context of a specific website. In that case, HCL DX provides inline editing capabilities that are integrated to the websites built on the platform. This enables content authors to browse to the relevant page in [site manager](site_building.md) and edit the content in context of the page where it is displayed.
+However, many people create content in the context of a specific website. In that case, HCL DX provides inline editing capabilities that are integrated with the websites built on the platform. With this approach,  content authors can browse to the relevant page in the [site manager](site_building.md) and edit the content in the context of the page where it is displayed.
 
-These two modes of content editing are not mutually exclusive. Since a single item of content may be presented in multiple different ways, changes made to the source are reflected wherever it is presented.
+These two modes of content editing are not mutually exclusive. Because on content item might be presented in multiple ways, changes to the source are reflected wherever it is presented.
```

Comment:
```suggestion
These two modes of content editing are not mutually exclusive. Because a content item might be presented in multiple ways, changes to the source are reflected wherever it is presented.
```

Reply:
@philip-menzies , committing this change. Let me know if OK with you.

Reply:
OK!